Content.Server/Shuttles/Systems/DockingSystem.cs

```diff
@@ -350,26 +350,26 @@ public void Dock(DockingComponent dockA, DockingComponent dockB)
             dockB.DockJoint = joint;
             dockB.DockJointId = joint.ID;
 
-            if (TryComp<AirlockComponent>(dockA.Owner, out var airlockA))
+            if (TryComp(dockA.Owner, out DoorComponent? doorA))
             {
-                airlockA.SetBoltsWithAudio(true);
+                doorA.ChangeAirtight = false;
+                _doorSystem.TryOpen(doorA.Owner, doorA);
             }
 
-            if (TryComp<AirlockComponent>(dockB.Owner, out var airlockB))
+            if (TryComp(dockB.Owner, out DoorComponent? doorB))
             {
-                airlockB.SetBoltsWithAudio(true);
+                doorB.ChangeAirtight = false;
+                _doorSystem.TryOpen(doorB.Owner, doorB);
             }
 
-            if (TryComp(dockA.Owner, out DoorComponent? doorA))
+            if (TryComp<AirlockComponent>(dockA.Owner, out var airlockA))
             {
-                doorA.ChangeAirtight = false;
-                _doorSystem.StartOpening(doorA.Owner, doorA);
+                airlockA.SetBoltsWithAudio(true);
```

Comment:
Shouldn't this check if the door successfully opened?

Also if we need ChangeAirtight then it should be handled upon undocking too.